### ISSUE TITLE
Model Builder Error "Value cannot be null. Parameter name: source"

### ISSUE THREAD

**Issue by ladipro** [dotnet/machinelearning-modelbuilder#1333]
**System Information (please complete the following information):**
 - Model Builder Version: 16.100.20.27905
 - Visual Studio Version: Version 16.10.0 Preview 2.0 [31110.410.main]

**Describe the bug**
Step 4 fails for me when trying to train Image classification.

```
   at System.Linq.Enumerable.Select[TSource,TResult](IEnumerable`1 source, Func`2 selector)
   at Microsoft.ML.ModelBuilder.AutoMLService.Experiments.AutoMLExperiment`3.<ExecuteAsync>d__21.MoveNext() in /_/src/Microsoft.ML.ModelBuilder.AutoMLService/Experiments/AutoMLExperiment.cs:line 118
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.ML.ModelBuilder.AutoMLEngine.<StartTrainingAsync>d__30.MoveNext() in /_/src/Microsoft.ML.ModelBuilder.AutoMLService/AutoMLEngineService/AutoMLEngine.cs:line 147
```

**To Reproduce**
- Scenario: Image classification
- Environment: Local
- Data: A folder on Desktop with three subfolders. The subfolders contain 9, 11, and 9 JPG images respectively.

Willing to share the training data. Please ping me.

**Expected behavior**
Training succeeds.

**Screenshots**
![image](https://user-images.githubusercontent.com/12206368/111625812-944ceb80-87ed-11eb-8fcd-1b9e4aec14f8.png)

**Additional context**
My first time using Model Builder. It's possible that I'm doing something wrong.

**Comment by beccamc** [comment]
@ladipro Hi Ladi! How's it going? Sorry you ran into this problem. Can you update to version 16.3 from [here](https://marketplace.visualstudio.com/items?itemName=MLNET.07&ssr=false#overview)? Also, you can ping me a zip of your images and I can give it a try on the newer version. Thanks!

**Comment by ladipro** [comment]
@beccamc Thank you Becca! Updating to 16.3 indeed fixed my issue. It's odd that the latest version is not included in the latest VS build but hey I'm unblocked now and can continue playing with ML.NET. Thanks again.

**Comment by pomeara** [comment]
@beccamc sure thing here are my nugets...

    <PackageReference Include="Microsoft.ML" Version="1.6.0" />
    <PackageReference Include="Microsoft.ML.FastTree" Version="1.6.0" />
    <PackageReference Include="Microsoft.ML.LightGbm" Version="1.6.0" />

And here is my stack trace
`Copying generated code to project... (Microsoft.ML.ModelBuilder.Utils.Logger.Info)
2021-10-30 03:39:43.1003 INFO Copying MLCheck.consumption.cs to folder: D:\xxx\ML.Project\ML (Microsoft.ML.ModelBuilder.Utils.Logger.Info)
2021-10-30 03:39:43.1003 INFO Copying MLCheck.training.cs to folder: D:\xxx\ML.Project\ML (Microsoft.ML.ModelBuilder.Utils.Logger.Info)
2021-10-30 03:39:43.1003 INFO COMPLETED (Microsoft.ML.ModelBuilder.Utils.Logger.Info)
2021-10-30 03:39:43.1003 INFO 

Updating nuget dependencies... (Microsoft.ML.ModelBuilder.Utils.Logger.Info)
2021-10-30 03:39:43.1003 INFO Installing nuget package, package ID: Microsoft.ML, package Version: 1.6.0 (Microsoft.ML.ModelBuilder.Utils.Logger.Info)
2021-10-30 03:39:43.1003 DEBUG Value cannot be null.
Parameter name: project
   at NuGet.VisualStudio.VsPackageInstallerServices.GetInstalledPackages(Project project)
   at Microsoft.ML.ModelBuilder.Utils.Utilities.<InstallNugetPackageAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.ML.ModelBuilder.ViewModels.TrainViewModel.<UpdateNugetDependenciesAsync>d__119.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Microsoft.ML.ModelBuilder.ViewModels.TrainViewModel.<GenerateCodeBehindFilesAsync>d__118.MoveNext() (Microsoft.ML.ModelBuilder.Utils.Logger.Debug)
`

**Comment by beccamc** [comment]
@pomeara Somehow the project we've selected is null. Do you have an unloaded project in your solution? Or a project in a bad state?

**Comment by beccamc** [comment]
I've opened a new issue to track this, as it's different than Ladi's original issue. https://github.com/dotnet/machinelearning-modelbuilder/issues/1903
